src/test/article.spec.js

```diff
@@ -1199,3 +1203,92 @@ describe('GET /api/v1/articles/popular', () => {
     findAllStub.restore();
   });
 });
+describe('POST api/v1/:articleId/comments/:commentId/like', () => {
+  beforeEach(async () => {
+    await db.Article.destroy({ cascade: true, truncate: true });
+    await db.User.destroy({ cascade: true, truncate: true });
+    // await db.Commentlike.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should like a comment', async () => {
+    const author = await createUser();
+    const commenter = await createUser();
+    const liker = await createUser();
+
+    const createdArticle = await createArticle(
+      await generateArticle({ authorId: author.id })
+    );
+
+    const createdComment = await createComment(
+      createdArticle.id,
+      await generateComment({ userId: commenter.id })
+    );
+    const { id: commentId } = createdComment[0].dataValues;
+
+    const token = helper.jwtSigner({ id: liker.id });
+    const res = await server()
+      .post(`/api/v1/${createdArticle.id}/comments/${commentId}/like`)
+      .set('token', token)
+      .send({ type: 'comment' });
+    expect(res.status).to.equal(201);
+    expect(res.body.data).to.be.equal('Comment liked');
+  });
+
+  it('should throw error if article id is invalid', async () => {
+    const liker = await createUser();
+
+    const token = helper.jwtSigner({ id: liker.id });
+    const res = await server()
+      .post(`/api/v1/234/comments/7/like`)
+      .set('token', token)
+      .send({ type: 'comment' });
+    expect(res.status).to.equal(400);
+    expect(res.body.message).to.be.equal(
+      'The specified comment id does not exist'
+    );
+  });
+
+  it('sshould unlike a comment for an article', async () => {
```

Comment:
The current project does not support features like `unlike`, users should only be able to remove there previous likes. Please amend this.

Reply:
but this branch support feature that makes an authenticated user to like or unlike a specific comment. the PT board story says "Users should be able to like a specific comment". 
if I have liked a specific comment and then later I am not interested in such comment anymore I should unlike it.